modules/benchmark/R/format_wide2long.R

```diff
@@ -0,0 +1,103 @@
+##' @name format_wide2long
+##' @title format_wide2long
+##' 
+##' @param out wide format data
```

Comment:
because it's the `out` coming from [load_data](https://github.com/PecanProject/pecan/blob/master/modules/benchmark/R/load_data.R#L55), I'll try to improve all my headers